lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -189,13 +183,12 @@ function dosomething_signup_sms_game_form($form, &$form_state, $node, $num_frien
     ),
   );
   for ($i = 0; $i < $num_friends; $i++) {
-    // Default to second fieldset.
-    $group = 'group2';
-    if ($i == 0) {
-      // Unless its the first beta.
-      $group = 'group1';
-    }
-    $form[$group]['beta_mobile_' .$i] = array(
+    $form['group1']['beta_first_name_' .$i] = array(
+      '#type' => 'hidden',
+      '#title' => 'Your Friend\'s Name',
```

Comment:
Happy to update anyways, but it's more of a developer convenience ("what is this for?") than anything. I don't even think Drupal creates `<label>` elements for hidden form fields.